resources/migrations/000_migrations.yaml

```diff
@@ -12085,6 +12085,72 @@ databaseChangeLog:
             columnName: parameter_mappings
             defaultNullValue: '[]'
             tableName: report_card
+  - changeSet:
+      id: v44.00-031
+      author: snoe
+      comment: Added 0.44.0 - writeback
+      changes:
+        - createTable:
+            tableName: api_action
```

Comment:
Would it maybe make more sense to call this `http_action` instead? API itself doesn't necessarily mean HTTP request, e.g. calling `String/indexOf` or whatever is using the `java.lang.String` API